android/app/src/main/java/com/chorestory/app/ParentCreateClanActivity.java

```diff
@@ -71,7 +90,6 @@ public void onClick(View v) {
                 clanName = clanNameEditText.getText().toString();
                 name = nameEditText.getText().toString();
                 password = passwordEditText.getText().toString();
-                picture = R.drawable.king_color; // TODO: Let the user select this when the option is created
```

Comment:
Shouldn't this be `picture = picture`?

Reply:
I set  `picture = R.drawable.king_color;` as default in onCreate, then it updates if user clicks on the other one. this is correct

Reply:
Okay, nevermind I see it!